### DISCUSSION TITLE
Unable to see any image pushed from local repo or images uploaded from github itself

### DISCUSSION THREAD

**Discussion by Sindhuk96** [community/community#79193] [Other Feature Feedback, Questions, & Ideas]
![gitHub](https://github.com/community/community/assets/119129390/7c65e955-cb1d-41a6-8f8b-33d00d0c48ed)
![gitHubDev](https://github.com/community/community/assets/119129390/a0b1b8f0-31bc-4e05-8ad0-a27681da397e)
### Select Topic Area

Bug

### Body

I have pushed my project images to my project repository in github. In my local repo i.e., on vs code i am able to view the images but in github it is taking some to load and broken-image is seen after that.

Issue is seen for all the images in my github.
Issue is also seen for images that I uploaded from github.
Issue is not seen for the images in github.dev

Please help me how to resolve this issue. I am a beginner and not getting how to resolve this issue.
Github url for img: https://github.com/Sindhuk96/petBarlo/blob/main/src/utils/assets/petBarloLogo.png

pls find the attached screenshots for GitHub and GitHub.Dev

**Comment by lbxxtt** [comment]
Hi @Sindhuk96, I'm running into the exact same issue as you, any update on how you resolve this issue?
